packages/x-date-pickers/src/internals/hooks/useField/useFieldState.ts

```diff
@@ -94,6 +106,7 @@ export const useFieldState = <
     return {
       sections,
       value: valueFromTheOutside,
+      placeholder: fieldValueManager.getValueStrFromSections(emptySections),
```

Comment:
We could generate the placeholder in the same memo as `emptySections`

Reply:
Done. Refactored to a `placeholder` memo. 😉